suite/tests/CMakeLists.txt

```diff
@@ -3013,6 +3013,10 @@ else (UNIX)
   tobuild(security-win32.except-guard security-win32/except-guard.c)
   tobuild(security-win32.thread-setcontext security-win32/thread-setcontext.c)
   tobuild(security-win32.singlestep security-win32/singlestep.c)
+  if (NOT X64)
+    # no way to set debug register in 64 bit
+    tobuild(security-win32.except-debugreg security-win32/except-debugreg.c)
+  endif (NOT X64)
```

Comment:
If it works on x64, could you add comment here about how to (manually) test it on x64 if possible, at least say the 32-bit limitation is test only.
If it does not work on x64, could you return false in debug_register_fire_on_addr for x64, and add comment in that function about x64 not working.

debug_register_fire_on_addr()
{
#ifndef X64
...
#endif
return false;
}